alpha1: 0.2
alpha2: 0.2
alpha3: 0.1
alpha4: 0.1
alpha5: 0.02
alpha6: 0.02
num_particles: 1000
sigma_r: 0.02
sigma_b: 0.01
gt_topic: /gazebo/model_states
